### PR TITLE
Adds contract tests to equipment

### PR DESCRIPTION
**Pull request by waldemarnt**
We are adding contract tests to cover the communication
between equipment façade and telemetry API equipment resource

### CODE REVIEW

File: contract_config.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+(process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0');
```
Comment (andressa) [inline contract_config.js]: What is the motivation of include this line between parenthesis?

File: contract_helper.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+'use strict'; // eslint-disable-line strict
```
Comment (bltavares) [inline contract_helper.js]: is this line needed?

**Comment by andressa** [comment]
👍

**Comment by jeffhsta** [comment]
:+1:

File: contract_helper.js (hunk 2 above)
Comment (waldemarnt) [inline contract_helper.js]: Are you right, it was a test that we did in a old version. Fixed here https://github.com/agco-fuse/fuse-equipment-api/pull/52/commits/98a881da9c32a797d55fbe28e5a2b138f8503db3

File: contract_config.js (hunk 1 above)
Comment (waldemarnt) [inline contract_config.js]: Good point, we don't need do that in this test. I just fixed here https://github.com/agco-fuse/fuse-equipment-api/pull/52/commits/18c70dd3ab5108ba4a728cbf26215ff594596bb1

File: contract_http_client.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import requestPromise from 'request-promise';
+import helper from './contract_helper';
+
+const contractHttpRequest = (request, cb) => {
+  const contractRequest = request;
```
Comment (bltavares) [inline contract_http_client.js]: What if we change the argument name to `contractRequest` instead?

Comment (waldemarnt) [inline contract_http_client.js]: i liked and did a commit here https://github.com/agco-fuse/fuse-equipment-api/pull/52/commits/80b9854a7c855a9f37b744946d48de40a7338ee2

Comment (waldemarnt) [inline contract_http_client.js]: Lint just broke, eslint expects that we can't change the `request` object , we need to assign to a new variable, i did a new commit to keep the same as before

File: .travis.yml [hunk 4]
```diff
@@ -10,6 +10,8 @@ addons:
     - g++-4.8
 env:
 - TRAVIS=travis CXX=g++-4.8
+after_script:
+- npm run contract-tests
```
Comment (jeffhsta) [inline .travis.yml]: What about we add contract-tests on the "test" script in our `package.json`, to just call `npm test` and run all tests we have?

**Pull request merged**